### PR TITLE
Unit Test Troubleshooting.

### PR DESCRIPTION
**Pull request by gilbertk88**

### CODE REVIEW

File: bin/push_to_wordpress_org.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ svn co \
 
 if [ -d "wp_org/tags/$PLUGIN_VERSION" ]; then
   echo "Version exist. Exiting."
-  exit 1
+  exit 12
```
Comment (vpetersson) [inline bin/push_to_wordpress_org.sh]: @gilbertk88 Why are you setting exit code 12?

Comment (gilbertk88) [reply bin/push_to_wordpress_org.sh]: I rolled it back to 1

File: .travis.yml [hunk 2]
```diff
@@ -59,7 +61,8 @@ script:
       phpunit
       WP_MULTISITE=1 phpunit
     fi
-  - |
-    if [[ "$WP_TRAVISCI" == "phpcs" ]] ; then
-  - test $TRAVIS_BRANCH = "master" && ./bin/push_to_wordpress_org.sh
-  
\ No newline at end of file
+   -if [ "$TRAVIS_BRANCH" = "master" ] || [ "$TRAVIS_PULL_REQUEST" = "false" ]; 
+        then./bin/push_to_wordpress_org.sh
```
Comment (vpetersson) [inline .travis.yml:65]: @gilbertk88 This is wrong. Should be:
```
if [[ statement ]]; then
    ./some-script.sh
fi
```

File: bin/push_to_wordpress_org.sh (hunk 1 above)
Comment (leyini) [reply bin/push_to_wordpress_org.sh]: Gilbert,
Kindly reach me. I had to check the web for a means to reach you with respect to the simplelender WP plugin. My email is : leyini2010@gmail.com. Apologies if am sending this in the wrong environment.

**Pull request merged**